contracts/MasterChefV2.sol

```diff
@@ -219,42 +218,39 @@ contract MasterChefV2 is BoringOwnable, BoringBatchable {
     /// @notice Harvest proceeds for transaction sender to `to`.
     /// @param pid The index of the pool. See `poolInfo`.
     /// @param to Receiver of SUSHI rewards.
-    function harvest(uint256 pid, address to) public {
+    function harvest(uint256 pid, address to) public returns (bool success, bytes memory) {
```

Comment:
Could you change the return as discussed to `(bool success)`?

Reply:
yeah that looks good, I made this change in latest edit 📋